grammar.js

```diff
@@ -690,23 +698,26 @@ module.exports = grammar({
       )),
       '"',
     ),
-    _multiline_string_literal: $ => seq(
+    _multiline_string_literal: $ => prec.right(seq(
       '"""',
       repeat(choice(
         alias($._multiline_string_content, $.string_content),
         $._escape_sequence,
       )),
       '"""',
-    ),
+    )),
 
     character: $ => seq(
       '\'',
-      choice(
-        token.immediate(prec(2, /[^'\\]/)),
-        $._escape_sequence,
+      repeat1(
+        choice(
+          $.character_content,
+          $._escape_sequence,
+        ),
       ),
       '\'',
     ),
+    character_content: _ => token.immediate(prec(2, /[^'\\]+/)),
```

Comment:
can characters be more than one char? that's weird, just double checking

Reply:
Yes, they can. And yes, it is weird. It will build up a numeric type with the single bytes of the numeric type being the characters in the literal. So `let foo: U64 = 'HTTP/1.1'` will become `0x485454502F312E31`. This same behavior exists in C as well.